### PR TITLE
Use <buffer> directive

### PR DESCRIPTION
**Pull request by kenhys**
Updated to v0.14 style configuration.

### CODE REVIEW

File: sample/command.conf [hunk 1]
```diff
@@ -10,5 +10,8 @@
   @type groonga
   protocol command
   database /tmp/groonga/db
+
+  <buffer>
   flush_interval 1
```
Comment (cosmo0920) [inline sample/command.conf]: Could you add an indentation like as https://github.com/groonga/fluent-plugin-groonga/pull/17/commits/beaa8634b8438eb2266fcd14fba11a6071957fbd?

File: sample/gqtp.conf [hunk 2]
```diff
@@ -11,5 +11,8 @@
   protocol gqtp
   host 127.0.0.1
   port 30043
+
+  <buffer>
   flush_interval 1
```
Comment (cosmo0920) [inline sample/gqtp.conf]: ditto.

File: sample/http.conf [hunk 3]
```diff
@@ -11,5 +11,8 @@
   protocol http
   host 127.0.0.1
   port 30041
+
+  <buffer>
   flush_interval 1
```
Comment (cosmo0920) [inline sample/http.conf]: ditto.

**Comment by kenhys** [comment]
Thanks. I've fixed it.

File: sample/store-syslog.conf [hunk 4]
```diff
@@ -51,9 +51,11 @@
   protocol http
   host 127.0.0.1
 
-  buffer_type file
-  buffer_path /var/spool/td-agent/buffer/groonga
-  flush_interval 1
+  <buffer>
+    @type file
+    buffer_path /var/spool/td-agent/buffer/groonga
```
Comment (kou) [inline sample/store-syslog.conf]: Can we remove `buffer_` prefix?

Comment (cosmo0920) [inline sample/store-syslog.conf]: Yes, we can.
In buffer directive, path is correct parameter: https://github.com/fluent/fluentd/blob/master/lib/fluent/plugin/buf_file.rb#L36

File: sample/store.conf [hunk 5]
```diff
@@ -9,7 +9,9 @@
   protocol http
   host 127.0.0.1
 
-  buffer_type file
-  buffer_path /tmp/buffer
-  flush_interval 1
+  <buffer>
+    @type file
+    buffer_path /tmp/buffer
```
Comment (cosmo0920) [inline sample/store.conf]: We can also remove `buffer_` prefix here.

File: sample/store-apache.conf [hunk 6]
```diff
@@ -42,9 +42,11 @@
   protocol http
   host 127.0.0.1
 
-  buffer_type file
-  buffer_path /tmp/buffer
-  flush_interval 1
+  <buffer>
+    @type file
+    buffer_path /tmp/buffer
```
Comment (cosmo0920) [inline sample/store-apache.conf]: ditto.

**Comment by kenhys** [comment]
Thanks, I've removed `buffer_` prefix.

**Comment by kou** [comment]
Merged!

**Pull request merged**